docs/DeveloperGuide.md

```diff
@@ -248,6 +248,21 @@ The following activity diagram summarizes the flow of when a user enters an edit
 
 * **Future Extension:** bobaBot can support multiple editing so user do not have to edit customers one by one.
 
+### \[Insert Numbering\] Increase/ Decrease feature
+The Increase/ Decrease feature is facilitated by `LogicManager`. The `IncreaseCommandParser` or `DecreaseCommandParser` parses the command arguments, and returns
+an `IncreaseCommand` or `DecreaseCommand` that is executed by the `LogicManager`.
+
+This feature is an extension to the above Edit feature to ease the process of editing Reward points of a Customer.
+
+**Below is a sample usage and how the increase/ decrease sequence behaves at each step.**
+
+1. User chooses the Customer he/ she wants to increase or decrease the Reward points for and enters the command `incr 100 e/test@gmail/com` or `decr 100 e/test@gmail/com`
```

Comment:
The current implementation is COMMAND_WORD + CHANGES + IDENTIFIER.

Perhaps it would be good to specify that this is different from the normal convention of COMMAND_WORD + IDENTIFIER + CHANGES? Could add like a Note: Take note that this is different from...